barman/backup.py

```diff
@@ -371,6 +371,27 @@ def get_last_backup_id(self, status_filter=DEFAULT_STATUS_FILTER):
         ids = sorted(available_backups.keys())
         return ids[-1]
 
+    def get_last_full_backup_id(self, status_filter=DEFAULT_STATUS_FILTER):
+        """
+        Get the id of the latest/last FULL backup in the catalog (if exists)
+
+        :param status_filter: The status of the backup to return,
+            default to DEFAULT_STATUS_FILTER.
+        :return string|None: ID of the backup
```

Comment:
```suggestion
            default to :attr:`DEFAULT_STATUS_FILTER`.
        :return str|None: ID of the backup
```

Adjusting sphinx syntax.